subsys/bluetooth/mesh/Kconfig

```diff
@@ -935,6 +935,17 @@ config BT_MESH_TX_BLOB_CHUNK_SIZE
 	  see also: BT_MESH_TX_SEG_MAX,
 	  and the maximum SDU a node can receive.
 
+config BT_MESH_TX_BLOB_CHUNK_SEND_INTERVAL
+	int "BLOB Client chunk send interval"
+	default 0
+	range 0 1500
```

Comment:
Related to the comment above. I think in some configurations it still may be possible to set big (e.g. 2, 3 or more seconds) interval which will work with big timeout base value. I'm not sure we need this artificial limitation. Better to say about relation between this interval timeout base value and timeouts.

Reply:
I'm fine with ranging it till up to INT_MAX. I'm still uncertain with `In a test environment with a @p timeout_base value of 10, setting the interval above 1500ms, timeouts should be expected.` sentence as this arises more questions like what other configuration was used, what exact environment was used, etc. Also, customers won't use this in sterile environment, so this doesn't really make sense for them.

Reply:
alright, then I'll just remove that part :)

Reply:
seems INT_MAX is not defined in that space. Let me see if I can find a way, otherwise will set the max as a decimal number.